zerver/webhooks/stripe/view.py

```diff
@@ -90,8 +90,11 @@ def default_body() -> str:
                     for key, value in object_['metadata'].items():
                         body += '\n{}: {}'.format(key, value)
         if resource == 'discount':  # nocoverage
-            body = default_body() + '. ({coupon})'.format(
-                coupon=linkified_id(object_['coupon'], lower=True))
+            body = 'Discount (**{coupon_name}**) {event} for {resource}.'.format(
```

Comment:
Let's just make the text 
`Discount {verbed} ([{coupon_name}](https://dashboard.stripe.com/coupons/{coupon_id}))`

This is already threaded under the customer, so we don't need it in the text.
Thanks!